File: public/dsa/two_pointers/dutch_national_flag/naive/dutch_national_flag.cpp
#include <bits/stdc++.h>
using namespace std;

void sortTheArray(vector<int> &A) {
    sort(A.begin(), A.end());
}

// tests-start
int main() {
    vector<int> A = {2, 2, 0, 1};
    sortTheArray(A);
    for (int v : A) {
        cout << v << " ";
    }
    cout << endl;
    return 0;
}
// tests-end

/*output
0 1 2 2
*/

File: public/dsa/two_pointers/dutch_national_flag/naive/dutch_national_flag.py

from typing import List


class Solution:
    def sortTheArray(self, A: List[int]) -> None:
        A.sort()


# tests-start
A = [2, 2, 0, 1]
sol = Solution()
sol.sortTheArray(A)
print(A)
# tests-end

'''output
[0, 1, 2, 2]
'''
